chat-commands.js

```diff
@@ -1961,6 +1961,7 @@ exports.commands = {
 		target = this.splitTarget(target, true);
 		let name = this.targetUsername;
 		let nextGroup = target;
+		if (Config.groups[nextGroup].roomonly || Config.groups[nextGroup].battleonly) return this.errorReply(`Group '${nextGroup}' does not exist as a global rank.`);
 		if (!Config.groups[nextGroup]) return this.errorReply("Group '" + nextGroup + "' does not exist.");
```

Comment:
I think the order of these two lines should be switched, so that if it doesn't exist it won't try to read data from it first